app/views/ExploreDeepContent/index.tsx

```diff
@@ -0,0 +1,503 @@
+        printPreviewMode,
+        onHidePrintPreviewClick,
+        onPrintClick,
+        headerActions,
+        endDate,
+        startDate,
+        onEndDateChange,
+        onStartDateChange,
+        onImageExportClick,
+        exportIdToDownload,
+        onPdfExportClick,
+        onExcelExportClick,
+        onFiltersChange,
+        totalProjects,
+        totalRegisteredUsers,
+        totalActiveUsers,
+        totalLeads,
+        totalAuthors,
+        totalPublishers,
+        totalEntries,
+        totalEntriesAddedLastWeek,
+        projectsByRegion: projectsByRegionFromProps,
+        projectCompleteTimeseries,
+        entriesCompleteTimeseries,
+        leadsCountByDay,
+        entriesCountByRegion,
+        topTenAuthors,
+        topTenPublishers,
+        topTenFrameworks,
+        topTenProjectsByUsers,
+        topTenProjectsByEntries,
+        topTenProjectsByLeads,
+        loadingMapAndLeadData,
+        exportCreatePending,
+        filters,
+        loading,
+    } = props;
+
+    const startDateString = formatDateToString(new Date(startDate), 'yyyy-MM-dd');
+    const endDateString = formatDateToString(new Date(endDate), 'yyyy-MM-dd');
+
+    const [representationType, setRepresentationType] = useState<Option['key']>('table');
+    //
+    // FIXME: Remove this after fixed in server
+    const projectsByRegion = useMemo(() => (
+        projectsByRegionFromProps?.filter(isDefined) ?? undefined
+    ), [projectsByRegionFromProps]);
+
+    const projectContentProps = useMemo(() => ({
+        projectsByRegion: projectsByRegion ?? undefined,
+        readOnly: printPreviewMode || isPublic,
+        endDate,
+        startDate,
+        onEndDateChange: !isPublic ? onEndDateChange : undefined,
+        onStartDateChange: !isPublic ? onStartDateChange : undefined,
+        projectFilters: filters,
+        isPublic,
+        completeTimeseries: projectCompleteTimeseries ?? undefined,
+    }), [
+        projectsByRegion,
+        printPreviewMode,
+        filters,
+        startDate,
+        endDate,
+        isPublic,
+        onEndDateChange,
+        onStartDateChange,
+        projectCompleteTimeseries,
+    ]);
+
+    const entriesContentProps = useMemo(() => ({
+        endDate,
+        startDate,
+        readOnly: printPreviewMode || isPublic,
+        onEndDateChange: !isPublic ? onEndDateChange : undefined,
+        onStartDateChange: !isPublic ? onStartDateChange : undefined,
+        completeTimeseries: entriesCompleteTimeseries ?? undefined,
+        leadsCountByDay: leadsCountByDay ?? undefined,
+        entriesCountByRegion: entriesCountByRegion ?? undefined,
+        loading: loadingMapAndLeadData,
+    }), [
+        loadingMapAndLeadData,
+        endDate,
+        startDate,
+        printPreviewMode,
+        isPublic,
+        onEndDateChange,
+        onStartDateChange,
+        entriesCompleteTimeseries,
+        leadsCountByDay,
+        entriesCountByRegion,
+    ]);
```

Comment:
Why separate out the props? Why not directly define variables for ProjectContent and PEntriesContent component